src/modules/reservePopup.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-underscore-dangle */
+// import displayComments from './displayComments.js';
+
+import openReserveWindow from './openReserveWindow.js';
+
+const handleReservePopup = async () => {
+  const movieUrl = 'https://api.tvmaze.com/schedule/web?date=2020-05-29';
+
+  const ReservePopup = document.querySelector('.reserve-popup');
+  ReservePopup.style.display = 'none';
+  const viewReservebtn = document.querySelectorAll('.view-reservation');
+
+  try {
+    const data = await (
+      await fetch(movieUrl, {
+        method: 'GET',
+      })
+    ).json();
+    Array.from(viewReservebtn).forEach((element) => {
+      element.addEventListener('click', () => {
+        const reserveId = element.getAttribute('reserve-id');
+        const movieObj = data.find(
+          // eslint-disable-next-line comma-dangle
```

Comment:
- [ ] For best practice reasons and since it is a common practice in real working environment, I suggest fixing this listing issue instead of disabling linter checks here. Kindly find a way to resolve this without obstructing linter checks :pray: